apps/wing/src/commands/upgrade.ts

```diff
@@ -1,26 +1,22 @@
 import debug from "debug";
+import { currentPackage } from "../util";
 
 const log = debug("wing:update");
-const PJSON = require("../../package.json");
 const DEFAULT_UPDATE_RATE = 1000 * 60 * 60 * 24; // 1 day
 
 /** Handles checking for toolchain updates */
 export async function checkForUpdates() {
-  if (!process.stdout.isTTY) {
-    log("not a TTY, skipping update check");
+  if (!process.stdout.isTTY || process.env.CI === "true" || currentPackage.version === "0.0.0") {
```

Comment:
Is it possible that `process.env.CI` is equal to `"1"` in some places? Maybe a more fuzzy check would be preferred.